nano/templates/accounting.tmpl

```diff
@@ -0,0 +1,94 @@
+<script type="text/javascript">
+    function markGreen(){
+      var toColor = document.getElementById('diffinput');
+      toColor.style.backgroundColor = "#DDFFDD";
+      var elem = document.getElementById("regid");
+      var body = $('body');
+      elem.value = body.data('urlParameters')["src"];
+	  toColor.focus();
+    }
+</script>
+<div class="itemGroup">
+    <div class="item">
+        Record to Present
+    </div>
+	<div class="item">
+		<div class="itemLabel">
+			Savings (Current Cycle)
+		</div>
+		<div class="itemContent">
+			{{:data.savings}}
+		</div>
+	
+	<div class="item">
```

Comment:
This div is inside the previous item div, which I'm guessing is the cause of the rendering oddities? I don't really know.